ru/techdocs/api/client/hoodie.md

```diff
@@ -0,0 +1,302 @@
+| ---------- |:------:|:---------------:|:--------:|
+| event        | String   | custom event identifier            | yes |
+| eventHandler | Function | Function handling triggered event. | yes |
+
+Hoodie информирует вас о различных событиях. В частности, когда объект создаётся, изменяется или удаляется из хранилища. Чтобы обработать эти события используется функция-хэндлер. Вы можете регистрировать также обработчики собственных событий и вызывать их вручную. Подробней об этом: [trigger](#trigger).
+
+##### Пример
+<pre><code>hoodie.store.on('event', function(createdTodo) {
+  console.log('Запись добавлена => ', createdTodo);
+});
+</code></pre>
+
+
+<a id="one"></a>
+### one()
+**version:**      *> 0.2.0*
+
+* Это одноразовый вариант [on](#on) и [off](#off). Событие обработается только один раз.
+
+<pre><code>hoodie.one('event', eventHandler);</code></pre>
+
+| параметр     | тип      | описание                           | обязательно |
+| ------------ |:--------:|:----------------------------------:|:-----------:|
+| event        | String   | имя события                        | да          |
+| eventHandler | Function | хэндлер                            | да          |
+
+
+<a id="off"></a>
+### off()
+**version:**      *> 0.2.0*
+
+* Отключает обработчики. Хэндлер больше не вызовется.*
+
+<pre><code>hoodie.off('event');</code></pre>
+
+| option     | тип   | описание     | необходимо |
+| ---------- |:------:|:---------------:|:--------:|
+| event      | String | имя события | да |
+
+<pre><code>hoodie.on('event', eventHandler);</code></pre>
+
+Если **hoodie.store.on** подписывается на событие, то  **hoodie.store.off** делает противоположное.
+
+##### Пример
+
+<pre><code>var todoStore = hoodie.store('todo');
+todoStore.on('todo:done', function(doneTodo, t) {
+  // Это никогда не вызовется
+});
+
+todoStore.on('todo:done', function(doneTodo, t) {
+  // Это также
+});
+
+// Это отписывает ото всех обработчиков
+todoStore.off('todo:done');
+
+todoStore.findAll().done(function(allTodos) {
+  todoStore.trigger('todo:done', allTodos[0], new Date());
+});
+</code></pre>
+
+
+<a id="trigger"></a>
+### trigger()
+
+**version:**      *> 0.2.0*
+
+*Вызывает указанное событие. Включая кастомные и предопределённые события.*
+
+<pre><code>hoodie.trigger('event', param, param, param ...);</code></pre>
+
+| option     | type   | description     | required |
+| ---------- |:------:|:---------------:|:--------:|
+| event      | String | custom event identifier.                                 | yes |
+| param      | Object | Detail information the event will pass to the listeners. | no |
+
+Раз уж вы можете подписываться на события с помощью [on](#on), этот метод позволяет вызывать ваши обработчики. Это касается как стандартных типов, описанных в [hoodie.store.on](/techdocs/api/client/hoodie.store.html#storeon), так и ваших собственных (кастомных) событий. Допустим, вы хотите вызвать событие, когда задача выполнена:
```

Comment:
But better to rephrase:

> Данный метод позволяет вызвать обработчики, подписанные на события с помощью метода on. Метод может вызвать как стандартые типы...

Btw, “данный” is better then “этот” in case of subject being documented.